GoogleForCreators/web-stories-wp
# Dashboard: Improve story count for authors

## Context



## Summary

Fix the counts, check if user can ability to view other posts.

## Relevant Technical Choices

## To-do



## User-facing changes



| Before | After |
| ----------- | ----------- |
| <img width="1904" alt="Screenshot 2021-09-22 at 15 38 01" src="https://user-images.githubusercontent.com/237508/134365417-39d62f38-6302-4bf6-b713-f2f22a5f81a6.png"> | <img width="1904" alt="Screenshot 2021-09-22 at 15 39 38" src="https://user-images.githubusercontent.com/237508/134365537-f1f36647-1b28-452f-bc3f-e1393367c296.png"> |
| <img width="1904" alt="Screenshot 2021-09-22 at 15 37 34" src="https://user-images.githubusercontent.com/237508/134365660-9a88c6c3-2897-4dc0-bdbd-18bca49614c9.png"> | <img width="1904" alt="Screenshot 2021-09-22 at 15 39 52" src="https://user-images.githubusercontent.com/237508/134365679-f303b577-4865-4eab-aa7d-181ae6a8a1fc.png"> |
| <img width="1904" alt="Screenshot 2021-09-22 at 15 37 48" src="https://user-images.githubusercontent.com/237508/134365758-b9c3a663-cde9-41ab-87d4-d8489980543d.png"> | <img width="1904" alt="Screenshot 2021-09-22 at 15 40 40" src="https://user-images.githubusercontent.com/237508/134365770-3d6d53af-b861-4b2a-90eb-dc169ee26722.png"> |

## Testing Instructions






This PR can be tested by following these steps:

1. Login as admin. 
2. Create a number of stories, with different post status, published, private, draft and scheduled. 
3. Login as author
4. Create a number of stories, with different post status, published, private, draft and scheduled. 
5. Go to dashboard, sense the number next to the filters match to the number stories displays. 
6. Ensure spinner is not shown. 
7. Login as admin. 
8. Go to dashboard, sense the number next to the filters match to the number stories displays. 
9. Ensure spinner is not shown. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6613
